edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss

```diff
@@ -1,7 +1,7 @@
 // Theme variables.
 
 //color
-$primary: #B70005;
+$primary: #A52238;
```

Comment:
This change will affect other PRs, why do you need to change this value? @alexjmpb